standards/application/relay-reliability.md

```diff
@@ -0,0 +1,150 @@
+---
+title: RELIABILITY-RELAY
+name: Reliability for Relay Protocol
+category: Standards Track
+tags: [reliability, application]
+editor: Kaichao Sun <kaichao@status.im>
+contributors:
+  - Richard Ramos <richard@status.im>
+---
+
+## Abstract
+
+[Relay Protocol](https://github.com/vacp2p/rfc-index/blob/main/waku/standards/core/11/relay.md) in Waku is efficient for routing messages, but there's no guarantee that a message will reach to its destination. For example, the receiver in a chat application may miss messages when network issue happens at either sender or receiver side. In general, a message in Waku network includes 3 status:
+
+- **outgoing**, the message is posted by its creator but no confirmations from other nodes yet
+- **sent**, the message is received by any other node in the network
+- **delivered**, the message is acknowledged by the receiver
```

Comment:
This is true of applications, but is "delivered" something that really has meaning on the Waku layer? Waku is a broadcast protocol - in other words, the aim is to deliver the message "to everyone" (with the application determining the intended recipient). To me it seems like in such a system you only have "send attempted" and "send acknowledged" with the latter being a variable parameter in terms of how many nodes in the broadcast group have acknowledged your message. This is also only a status that makes sense from the publisher's point of view.

Reply:
>  The states and terms used here is trying to mimic from app's perspective, so it can be better understandable by app devs.

It is OK to have a transport spec contextualise itself in terms that make sense for typical app developers, but it needs to be explicit about separation of concerns here. Store queries and responses on their own cannot know if an app's messages are "delivered".

> state changes from receiver side

Indeed. My point is simply to make it explicit that we're referring to state from sender POV.

Reply:
I still think "delivered" here changes the agents suddenly from sender/receiver to sending application instance and receiving application instance. Perhaps OK to just phrase a bit more clearly: "the message is acknowledged on the application layer by the intended recipient"